src/Data/Group/Permutation.hs

```diff
@@ -89,3 +92,17 @@ pairwise p = (to p, from p)
 ($-) :: Permutation a -> a -> a
 ($-) = from
 {-# inline ($-) #-}
+
+-- | Embed a 'Group' into the 'Permutation' group on it's underlying set.
+--
+embed :: (Group g) => g -> Permutation g
```

Comment:
Let's make this a finite group, and we can add a unidirectional pattern synonym called `Cayley` for the embedding (and potentially make it bidi for the retract?)

Reply:
Infinite symmetric groups require a bit more structure (namely, you can talk about *subgroups* of the Sym(X) for any X, where those subgroups are cofinal w.r.t. to a set of fixed points). Either way, I'm fine with talking about general groups here. Not gonna die on that hill 😄